storage/redis_adapter.go

```diff
@@ -77,7 +161,12 @@ func (a *redisAdapter) Destroy(app string) error {
 	return nil
 }
 
+// Reopen the storage adapter-- in the case of this implementation, a no-op
 func (a *redisAdapter) Reopen() error {
-	// No-op
 	return nil
 }
+
+// Stop the storage adapter
+func (a *redisAdapter) Stop() {
+	a.stopCh <- true
```

Comment:
it'd be better to just close this channel rather than send on it. this line makes only 1 goroutine receive `true`, while the close makes all listening goroutines receive `false` (the zero value of `bool`). the latter is better because it broadcasts the stop event to all listeners.